### PR TITLE
Modifying LDAP parameters in docker-compose.yml to be read from env.config.sh

### PR DESCRIPTION
**Pull request by mihail-dev**
Hi,

In our case we are spinning up ADOP and configuring it to use our existing ldap instead of adop-ldap. We would like to keep our ldap configuration separately to the docker compose file in env.config.sh. This modification will be beneficial because:
- It makes upgrading ADOP to a newer version easier in the future.
- it is simpler and easier to maintain.

I have tested the change on AWS and everything functions properly.

Thanks.

### CODE REVIEW

**Comment by anton-kasperovich** [comment]
I agree with @mihail-dev points and happy to merge it, but could you please remove your second commit 5aa6045 from this PR ?

**Comment by mihail-dev** [comment]
@anton-kasperovich yep, well spotted, just removed the second commit.

File: docker-compose.yml [hunk 1]
```diff
@@ -181,13 +181,13 @@ sonar:
   environment:
     - "dependency:container==sonar-mysql"
     - "LDAP_URL=ldap://ldap:389"
```
Comment (anton-kasperovich) [inline docker-compose.yml]: @mihail-dev Found this place where you could also use ${LDAP_SERVER} variable right?

File: docker-compose.yml [hunk 2]
```diff
@@ -314,7 +315,7 @@ ldap-ltb:
     - "80"
   environment:
     LDAP_LTB_URL: "ldap://ldap:389"
```
Comment (anton-kasperovich) [inline docker-compose.yml]: ${LDAP_SERVER} variable?

**Comment by anton-kasperovich** [comment]
Build failed, because of Jenkins and looks like it's related to this PR, i ran it again, let's see on results, if it would fail again i will try this changes locally to have better understanding where's issue.

**Comment by anton-kasperovich** [comment]
Looks like it was some issue with build on Travis, which isn't related to this PR. Successfully deployed ADOP with PR changes. Thank you @mihail-dev . LGTM.

**Pull request merged**